src/Prototype/Runtime.hs

```diff
@@ -88,3 +108,21 @@ newtype AppM (mode :: AppMode) a
 instance L.MonadLog AppName (AppM mode) where
   askLogger = asks _rLogger
   localLogger f = local $ over rLogger f
+
+-- | Boot in the Stm storage mode. 
+bootStm :: MonadIO m => Conf -> m (Either RuntimeErr StmRuntime)
+bootStm _rConf@Conf {..} = do
+  -- Start off by creating a logger first; we may want to log steps in the boot process.
+  _rLogger <- createLogger
+  L.runLogT' _rLogger . bootEnv $ do
+    -- Instantiate storage 
+    L.info "Instantiating storage"
```

Comment:
I usually write my logging sentences with a final dot, except in this kind of `"Xxxing ..."` case where I use three dots. The implicit convention is that `"Xxxing ..."` precedes the actual operation being described, and e.g. `"Storage instanciated."` follows. What do you think ?

Reply:
Yeah; this is a good way to go about it. I'll fix the logging statements. 
I'll do these changes in the rebased branch.